findhelp/models.py

```diff
@@ -66,6 +66,10 @@ def __str__(self):
         return self.name
 
 
+def is_lnglat_in_nyc(lnglat: Tuple[float, float]) -> bool:
+    return Borough.objects.filter(geom__contains=Point(*lnglat)).exists()
```

Comment:
Any particular reason to check whether any of the boroughs include the point  (5 membership comparisons) rather than doing a union geometry of all the boroughs first and then doing 1 membership comparison?

I'm not positive which is faster and it's unlikely it matters since the number of boroughs in NYC is not changing very quickly (Jersey City for 6th boro!), but I'm just curious. Is it because that isn't as supported by Django's Queryset API? https://docs.djangoproject.com/en/3.2/ref/contrib/gis/geoquerysets/#contains

Reply:
Hmm yeah I think it's just because it'd be a more complex query?  Even if it is possible to pre-union them, it didn't occur to me to do so...  that said, it's also possible that if this ends up being inefficient, we might want to use the _county_ data instead of the _borough_ data, because I think the county data doesn't include shorelines, which the borough data (IIRC) does.  The Borough shapefiles are actually relatively large, and I think it's because of those shorelines, but maybe I am wrong... anyways, we can always look further into it if this proves to be a bottleneck!